### PR TITLE
nitpick: decrease cognitive complexity in buildah-bud

### PR DESCRIPTION
**Pull request by boaz1337**
**NOTE**: this is another suggestion I have. It doesn't mean I don't like the coding style - it's just my own opinion and if you think it's not vital feel free to close it. :smile: 

A small nitpick to `cmd/buildah/bud.go`.

from:
```
if cliArgs > 0 {
  ... // 5 LOC
} else {
  return error
}
```

to:
```
if cliArgs == 0 {
  return error
}
... // 5 LOC
```

The latter style makes the code flow better than with if-else (IMHO).

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
I like the change.

**Comment by rhatdan** [comment]
@TomSweeneyRedHat @pixdrift @vrothberg @umohnani8 PTAL

**Comment by giuseppe** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
bot, retest this please.

File: cmd/buildah/bud.go [hunk 1]
```diff
@@ -82,33 +82,32 @@ func budCmd(c *cli.Context) error {
...
 	} else {
-		return errors.Errorf("no context directory or URL specified")
+		// Nope, it was local.  Use it as is.
+		absDir, err := filepath.Abs(cliArgs[0])
+		if err != nil {
+			return errors.Wrapf(err, "error determining path to directory %q", cliArgs[0])
+		}
+		contextDir = absDir
 	}
+	cliArgs = cliArgs.Tail()
```
Comment (TomSweeneyRedHat) [inline cmd/buildah/bud.go]: nit of a nit, I'd prefer a line break after this line.

**Comment by TomSweeneyRedHat** [comment]
LGTM once tests are happy.  Looks like a registry network flake.

File: cmd/buildah/bud.go (hunk 1 above)
Comment (boaz1337) [reply cmd/buildah/bud.go]: No problem :+1:

**Comment by vrothberg** [comment]
LGTM as well
@rh-atomic-bot r=TomSweeneyRedHat

**Pull request closed without merging**